GitUI/CommandsDialogs/BrowseDialog/GitStatusMonitor.cs

```diff
@@ -460,14 +477,20 @@ private void Update()
                         try
                         {
                             GitExtUtils.ArgumentString cmd = GitCommandHelpers.GetAllChangedFilesCmd(excludeIgnoredFiles: true, UntrackedFilesMode.Default, noLocks: noLocks);
+                            cancelToken.ThrowIfCancellationRequested();
```

Comment:
Yes, I believe so. As I said, these tests lock up for me locally because these pending operations don't get cancelled when the whole monitor is being disposed.

Reply:
With #11194 this likely won't be necessary